src/main/resources/org/jenkinsci/plugins/docker/workflow/Docker.groovy

```diff
@@ -75,11 +75,17 @@ class Docker implements Serializable {
         new Image(this, id)
     }
 
+    String shell() {
+        node {
+            script.isUnix() ? "sh" : "bat"
```

Comment:
Would it be better to use `powershell` as the default for Windows based containers?

Reply:
powershell or do you want pwsh? 😆

Reply:
Actually, it looks like the `powershell` step currently only supports `powershell.exe` not `pwsh.exe`, so sticking with `bat` is the best option.